config/v1/types_build.go

```diff
@@ -72,6 +76,27 @@ type ImageLabel struct {
 	Value string `json:"value,omitempty"`
 }
 
+type RegistriesList []string
+
+type RegistriesConfig struct {
```

Comment:
@bparees you prefer `RegistriesConfig` over `RegistryConfig`?  the plural sounds weird to me as a struct name

Reply:
i don't care.

Reply:
(but it is the configuration for the behavior w/ respect to "registries" not one "registry"

Reply:
The underlying file where some of this data is stored is `/etc/containers/registries.conf`, so that's where the name is drawn from.